### PR TITLE
Rack of Labs support in eden commands

### PR DESCRIPTION
**Pull request by Insei**
Signed-off-by: Aleksandrov Dmitriy <goodmobiledevices@gmail.com>

### CODE REVIEW

File: cmd/edenRol.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+		assignCobraToViper(cmd)
+
+		client, err := rolgo.NewClient()
+		if err != nil {
+			log.Fatalf(err.Error())
+		}
+		if rolIPXEUrl == "" {
+			certsEVEIP = viper.GetString("adam.eve-ip")
+			eServerPort := viper.GetString("eden.eserver.port")
+			rolIPXEUrl = fmt.Sprintf("http://%s:%s/%s/%s/ipxe.efi.cfg", certsEVEIP, eServerPort, "eserver", configName)
```
Comment (giggsoff) [inline cmd/edenRol.go:62]: Seems we do not need this line

File: cmd/edenRol.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+var createRentCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create a new device rent",
+	Long:  `Create a new device rent`,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		assignCobraToViper(cmd)
+
+		client, err := rolgo.NewClient()
```
Comment (giggsoff) [inline cmd/edenRol.go:55]: Do we need some token or password to access the backend?

**Review by giggsoff — CHANGES_REQUESTED** [review]
Probably I miss something, but I cannot see IP address we try to access (where backend is located). Also it is nice to have small description (in docs) about new commands (about RoL in general and about steps to prepare device and expected results of commands we run).

File: cmd/edenRol.go (hunk 2 above)
Comment (Insei) [reply cmd/edenRol.go:55]: Yep, in ENV, we need to set ROL_API_URL and ROL_API_KEY.

File: cmd/edenRol.go (hunk 1 above)
Comment (Insei) [reply cmd/edenRol.go:62]: Yep, my misstake.

**Comment by Insei** [comment]
> Probably I miss something, but I cannot see IP address we try to access (where backend is located)
We do not yet have a static IP or DNS name, this address can be set using the ROL_API_URL environment variable.

**Review by giggsoff — APPROVED** [review]
LGTM

**Pull request merged**